src/pages/[platform]/build-a-backend/data/custom-subscription/index.mdx

```diff
@@ -179,7 +179,7 @@ const schema = a.schema({
     .handler(a.handler.custom({
       entry: "./publish.js"
     }))
-    .authorization([a.allow.private()]),
+    .authorization(allow => [allow.authenticated()]),
```

Comment:
It looks like all of this text is in a comment though, but just to have it match, shouldn't there be another change on line 189?

From https://github.com/aws-amplify/docs/pull/7252/files#diff-055f3927edf8d3749a150a7fe3010aaf5f503e8e27c06f8df3025b2e96576e04


![image](https://github.com/aws-amplify/docs/assets/54393192/942f429c-ec23-46d5-9836-a8e0e13f7442)